hare3/types.go

```diff
@@ -40,7 +40,7 @@ type IterRound struct {
 
 // Delay returns number of network delays since specified iterround.
 func (ir IterRound) Delay(since IterRound) int {
```

Comment:
How about instead avoid type casting (return uint32) and just check if `since` > `ir`:
```go
func (ir IterRound) Delay(since IterRound) uint32 {
	if ir.Absolute() > since.Absolute() {
		return ir.Absolute() - since.Absolute()
	}
	return 0
}
```